presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraRecordSink.java

```diff
@@ -74,14 +76,39 @@ public CassandraRecordSink(CassandraOutputTableHandle handle, CassandraSession c
         columnTypes = handle.getColumnTypes();
     }
 
+    @Inject
+    public CassandraRecordSink(CassandraInsertTableHandle handle, CassandraSession cassandraSession)
+    {
+        this.fieldCount = requireNonNull(handle, "handle is null").getColumnNames().size();
+        this.cassandraSession = requireNonNull(cassandraSession, "cassandraSession is null");
+
+        String schemaName = handle.getSchemaName();
+        StringBuilder queryBuilder = new StringBuilder(String.format("INSERT INTO \"%s\".\"%s\"(", schemaName, handle.getTableName()));
+        queryBuilder.append(String.join(",", handle.getColumnNames()));
```

Comment:
This is basically a copy-paste of inefficient and error prone code. Would you mind to rebase on top of this commit? https://github.com/arhimondr/presto/commit/c0c727ec7e3d515d0c1938c9d95d1fc2aae3f759